addon/helpers/split-grid-cell.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+
+/**
+ * @module helpers
+ */
+
+/**
+ * Lookup and output a split-grid row's cell value based on column's path
+ *
+ * @function split-grid-cell
+ * @param    {object} options - The Ember-supplied bound options representing the calling view's state
+ * @returns
+ */
+export default function( options ) {
+    var column       = options.data.keywords.column,
+        row          = options.data.keywords.row,
+        templateName = Ember.get( column, 'template' ),
+        valuePath    = Ember.get( column, 'valuePath' );
+
+    if ( valuePath ) {
+        return new Ember.Handlebars.SafeString( Ember.get( row, valuePath ) );
+    }
+
+    options.contexts.push( options.data.keywords.view );
+    options.types[ 0 ] = 'STRING';
+
+    Ember.Handlebars.helpers.render.call( this, templateName, 'row', options );
+}
```

Comment:
@ryanmarc What is your input regarding the structure of this file as it relates to our coding guidelines?  Not certain we have a specific example for this case but if memory serves our general sense in a situation like this would be that we would prefer there to be an else condition that lines 24-27 are contained in so that the return logic on line 21 is not as easily visually lost when reviewing the code flow.

If it is determined that such a change is needed in this file we should also add this specific instance as an example in our coding style guidelines.